Firestore/core/src/firebase/firestore/model/field_path.cc

```diff
@@ -30,9 +30,6 @@ namespace model {
 
 namespace {
 
-// TODO(varconst): move to C++ equivalent of FSTDocumentKey.{h,cc}
-const char* const kDocumentKeyPath = "__name__";
```

Comment:
Looking at other SDKs:

* each SDK has its own place to store this constant;
* this constant is only used in FieldPath and inside tests.

So it seemed easiest to just move it to the point of usage. Please let me know if there's a reason to have it inside DocumentKey/make it a global variable/etc.

Reply:
Feel free to make organizational changes but when you notice inconsistencies like these please fix them :-).

Reply:
To be fair, there seem to be reasons for this inconsistencies. E.g., Java can't have a global variable, so the variable is wrapped into a class, whereas Typescript even has a nice syntax for exporting variables.